node/client/src/lib.rs

```diff
@@ -70,7 +70,6 @@ pub trait RuntimeApiCollection:
 	+ sp_consensus_grandpa::GrandpaApi<Block>
 	+ sp_block_builder::BlockBuilder<Block>
 	+ frame_system_rpc_runtime_api::AccountNonceApi<Block, AccountId, Index>
-	+ pallet_transaction_payment_rpc_runtime_api::TransactionPaymentApi<Block, Balance>
```

Comment:
Curious to know why we're removing `Block` and `Balance`

Reply:
We are removing neither. We are only removing the transaction payment, as it is not required for a feeless implementation.